bedrock-devnet/devnet/genesis.py

```diff
@@ -8,7 +8,7 @@
 
 GENESIS_TMPL = {
     'config': {
```

Comment:
Ideally we can remove this template in favor of a command in OP Node that builds the L1 genesis file, then we have 1 source of truth for genesis files which is the op node `genesis` command